README.md

```diff
@@ -5,3 +5,12 @@ Installers for MNE-Python for macOS, Windows, and Linux.
 <img src="https://mne.tools/dev/_static/mne_installer_macOS.png" alt="Installer running on macOS" width="450px">
 
 Please visit [the installers section of the MNE documentation](https://mne.tools/dev/install/installers.html) for instructions on how to use them.
+
+## Development
+
+Locally, installers can be built using `tools/build_local.sh`. This has only been tested on Linux and macOS M1.
+
+Requirements:
+- `conda` with `constructor` (`conda install -c conda-forge constructor`)
+
+      WARNING: Your constructor will be patched on macOS.
```

Comment:
This is not entirely correct, the requirements are listed here:

https://github.com/mne-tools/mne-installers/blob/c98e645690356ef3c23f1ff61c982b2db04e4744/environment.yml#L3-L11

We rely on a fork of `constructor` and `menuinst`

Reply:
... and then on top of that there is a `patch` call, right?

Reply:
yes 😅😅😅😅😅